### PR TITLE
BE: MailJet Service  added to send mail

### PR DESCRIPTION
**Pull request by DipuBala**

### CODE REVIEW

File: frontend/utils/constants.ts
Comment (SeefatHimel) [inline frontend/utils/constants.ts:1]: why this change ?

Comment (DipuBala) [reply frontend/utils/constants.ts:1]: I added one of my emails. When I hit save full file changed.
